### ISSUE TITLE
Login issue in the middle of run??

### ISSUE THREAD

**Issue by naurasd** [DominikBuchner/BOLDigger2#21]
Hi Dominik,

using boldigger 2.0.3, python 3.12.1

For some reason, after running for a few hours, I get a weird error regarding login, see below:

```
Downloading data:   5%|▌         | 800/14588 [2:40:05<45:59:07, 12.01s/it]
Traceback (most recent call last):
  File "/home/naurasd/.local/lib/python3.12/site-packages/boldigger2/id_engine_coi.py", line 577, in main
    download_dataframe = gather_download_links(
                         ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/naurasd/.local/lib/python3.12/site-packages/boldigger2/id_engine_coi.py", line 149, in gather_download_links
    raise BadResponseError
boldigger2.exceptions.BadResponseError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/naurasd/.local/bin/boldigger2", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/naurasd/.local/lib/python3.12/site-packages/boldigger2/__main__.py", line 88, in main
    id_engine_coi.main(
  File "/home/naurasd/.local/lib/python3.12/site-packages/boldigger2/id_engine_coi.py", line 661, in main
    session, username, password = login.bold_login(
                                  ^^^^^^^^^^^^^^^^^
  File "/home/naurasd/.local/lib/python3.12/site-packages/boldigger2/login.py", line 54, in bold_login
    text_fields = [field.text for field in navigation_bar.find_all("a")]
                                           ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'find_all'
```

Just a temporary problem from BOLD's side?

Nauras

**Comment by DominikBuchner** [comment]
Maybe they changed the login. I can check this on Sunday, too many things
happening at bolds end at the moment

naurasd ***@***.***> schrieb am Fr., 2. Aug. 2024, 09:30:

**Comment by DominikBuchner** [comment]
You can maybe change to v. 2.0.4 to adjust to the new download level and
see if the issue persists

Dominik Buchner ***@***.***> schrieb am Fr., 2. Aug.
2024, 10:08:

**Comment by DominikBuchner** [comment]
And the engine seems to be down again as well

Dominik Buchner ***@***.***> schrieb am Fr., 2. Aug.
2024, 10:09:

**Comment by naurasd** [comment]
alright, thanks. will try with new version!

**Comment by DominikBuchner** [comment]
Fixed with 2.1.0
